src/config/index.ts

```diff
@@ -51,6 +51,7 @@ export type ContactProperty = {
   type: ContactPropertyType;
   required: boolean;
   parameter? : string | string[] | object;
+  friendly_country_name?: string;
```

Comment:
can we keep config simple by using `not a valid KE phone number` instead of Kenya?

Reply:
Sure... I added that because the issue description suggested Kenya instead of KE. I'll revert to use KE

Reply:
Sorry to give conflicting suggestions. I see that in the issue now. I personally doubt it is worth the complexity in config.  Does libphonenumber-js know that KE means Kenya?

> Cannot find 'c_community_health_unit' matching Nakaale under Turkana South  should be Cannot find 'community health unit' matching Nakaale under Turkana South

If this isn't fixed, I'd recommend not yet closing #149 with this PR.